tests/network/network_simplification/test_network_graph_simplificator.py

```diff
@@ -276,6 +302,48 @@ def _get_expected_result_graph_fixture(
             ),
         )
 
+        _result_digraph.add_edge(
+            16,
+            2.0,
+            a='None',
+            from_node=16,
+            to_node=2,
+            geometry=LineString(
+                [
+                    _nx_digraph.nodes[16]["geometry"],
+                    _nx_digraph.nodes[1]["geometry"],
+                    _nx_digraph.nodes[2]["geometry"],
+                ]
+            ),
+        )
+
+        _result_digraph.add_edge(
+            16,
+            17,
+            a='None',
+            from_node=16,
+            to_node=17,
+            geometry=LineString(
+                [
+                    _nx_digraph.nodes[16]["geometry"],
+                    _nx_digraph.nodes[17]["geometry"],
+                ]
+            ),
+        )
+
+        _result_digraph.add_edge(
+            16,
+            18,
+            a='None',
+            from_node=16,
+            to_node=18,
+            geometry=LineString(
+                [
+                    _nx_digraph.nodes[16]["geometry"],
+                    _nx_digraph.nodes[18]["geometry"],
+                ]
+            ),
+        )
```

Comment:
Quite some repetitive code. This can be slimpified by a function, I guess.

Reply:
add_edge_with_attributes added